혜민/BOJ\_13300.py

```diff
@@ -0,0 +1,31 @@
+from bisect import bisect_right, bisect_left
+
+def count_0_1(a,right,left):
+	right_index = bisect_right(a,right)
+	left_index = bisect_left(a,left)
+	return right_index-left_index
+
+n, k = map(int, input().split())
+
+graph=[[] for _ in range(7)]
+
+#간선정보 입력 받기 
+for _ in range(n):
+	s, y = map(int, input().split())
+	graph[y].append(s)
+
+count = 0 
+for i in range(1,7):
+	graph[i].sort()
+	index_0 = count_0_1(graph[i],0,0)
+	index_1 = count_0_1(graph[i],1,1)
+	if index_0%k == 0:
+		count+=(index_0//k)
+	else:
+		count+=(index_0//k+1)
+	if index_1%k == 0:
+		count+=(index_1//k)
+	else:
+		count+=(index_1//k+1)
```

Comment:
p4: 해당 부분을 간단하게 더 줄여볼 수 있을 것 같아요

아이디어만 구상하고 로직은 지피티 돌린거라 틀렸을수도 있어요 ㅎㅎ..

```suggestion
	count += (round(index_0 / k) + round(index_1 / k)) 
```